hack/config.sh

```diff
@@ -61,4 +61,4 @@ kubectl="${_cli} --prefix ${PROVIDER} ssh node01 -- sudo kubectl --kubeconfig=/e
 DEPLOYMENT_TIMEOUT=600
 USE_CSI=${USE_CSI:-1}
 USE_RESTIC=${USE_RESTIC:-0}
-CSI_PLUGIN=${CSI_PLUGIN:-velero/velero-plugin-for-csi:main}
+CSI_PLUGIN=${CSI_PLUGIN:-velero/velero-plugin-for-csi:v0.2.0}
```

Comment:
isn't main like latest? i.e is it not being updated to be the latest version?

Reply:
Yes it was latest because @tomob did some PR that were only available in latest. Now as 0.2.0 (https://github.com/vmware-tanzu/velero-plugin-for-csi/releases/tag/v0.2.0) was released we can use it. And I would like to use a released version in e2e tests to have the builds repeatable.